ownership-chain/pallets/parachain-staking/src/lib.rs

```diff
@@ -119,7 +120,7 @@ pub mod pallet {
 
 	/// Configuration trait of this pallet.
 	#[pallet::config]
-	pub trait Config: frame_system::Config {
+	pub trait Config: frame_system::Config + pallet_block_rewards_source::Config {
```

Comment:
IF we are going with defining storage for reward account in a separate pallet, then this doesn't make that much sense, IMO. We are tightly coupling two pallets, and the other pallet has nothing except one storage

also, is this used anywhere in the code, like pallet_block_rewards::RewardsAccount::<T>::get()?

Reply:
we need because is used here: https://github.com/freeverseio/laos/blob/20af53a43dee40cd0de5daeb9e7f473af44ccfa8/ownership-chain/runtime/laos/src/configs/parachain_staking.rs#L164-L165

Reply:
but that's on a runtime level. You can simply add a trait bound in the `PayoutReward` trait:

```rs
impl<Runtime: StakingConfig + BlockRewardsSourceConfig> PayoutReward<Runtime> ...`
```